war/src/main/less/form/search.less

```
@@ -135,3 +135,29 @@
     }
   }
 }
+
+.close-icon {
+	border:1px solid transparent;
+	background-color: transparent;
+	display: block;
+	vertical-align: middle;
+  outline: 0;
+  cursor: pointer;
+}
+
+.close-icon:after {
+	content: "x";
```

I think [U+1F5D9 CANCELLATION X](https://www.fileformat.info/info/unicode/char/1f5d9/index.htm) is a better Unicode glyph for this character than [U+0078 LATIN SMALL LETTER X](https://www.fileformat.info/info/unicode/char/0078/index.htm).

```suggestion
	content: "\1F5D9";
```

(+ whatever other CSS changes are necessary to vertically align the new glyph in the middle)